build/.vsts-ci.yml

```diff
@@ -79,6 +79,8 @@ jobs:
       IOTHUB_CONN_STRING_INVALIDCERT: $(IOTHUB-CONN-STRING-INVALIDCERT)
       DPS_GLOBALDEVICEENDPOINT_INVALIDCERT: $(DPS-GLOBALDEVICEENDPOINT-INVALIDCERT)
       PROVISIONING_CONNECTION_STRING_INVALIDCERT: $(PROVISIONING-CONNECTION-STRING-INVALIDCERT)
+  - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+    displayName: 'Component Detection'
   - script: cd .. && rd /Q /S $(Agent.BuildDirectory)\s
```

Comment:
we don't need to clean up build with 1ES images. The trashcan still needs the cleanup.

Reply:
Cool, I'm updating the other jobs to follow that

Reply:
Should be just OSX and XCode Native that need the cleanup step!

Reply:
Done.

Reply:
Cleanup steps for windowsdynamic, windowsx64, and windowsx86 should be removed as well